server/auth/testing_utils.go

```diff
@@ -0,0 +1,37 @@
+package auth
```

Comment:
so I had to do this because this code would not be exported outside the package.
Since `NewTestAccessManager` is implementing the `AccessManager` interface and is used only in tests I will use mocks for this test and removed the `NewTestAccessManager`.That's why I've moved this testing utility in another folder called testing_utils.go 
Actually what I'm doing currently is